### DISCUSSION TITLE
multi-choice parameters

### DISCUSSION THREAD

**Discussion by y-j-huang** [jspsych/jsPsych#1415] [Q&A] [answered]
I'm trying to create a study with multi-choice questions but I have a hard time figuring out the parameters for it. It seems that either my practice_info or practice_trial or both doesn't work. I would appreciate it if someone could let me know what went wrong. Thank you!

var timeline = []; 

	var practice_info = [{questions: [{prompt: "I like vegetables", name: 'Vegetables', options: ["Yes", "No"], required: true}]},
	{questions: [{prompt: "I like fruits", name: 'Fruits', options: ["Yes", "No"], required: true}]}
	];
    
    var practice_trial = {
        type: 'survey-multi-choice',
        questions: jsPsych.timelineVariable('questions'),
        prompt: jsPsych.timelineVariable('prompt'),
        options: jsPsych.timelineVariable('options'),
        required: jsPsych.timelineVariable('required'),
        data: {trial_part: practice}
    };
    
    var practice = {
        timeline: practice_trial
        timeline_variables: practice_info,
    }
    
	timeline.push(practice);

**Comment by Nega100** [comment, marked as answer]
Hi @y-j-huang 
if I understand the code correctly you might forget to add the [ ] in timeline practice_trial array. 
```
var timeline = [];
var practice = {
    timeline: [practice_trial],
    timeline_variables: practice_info,
}
```

**Reply by y-j-huang** [reply]
Thank you so much!
